thinc/layers/tensorflowwrapper.py

```diff
@@ -1,10 +1,19 @@
-from typing import Any, Callable, Optional, Tuple, Type, TypeVar
+from typing import Any, Callable, Dict, List, Optional, Tuple, Type, TypeVar
 
+import numpy as np
```

Comment:
Also nitpick, but I think we don't use the `np` shortcut in the code base and just use `numpy` as `numpy`.